bucket/fun.json

```
@@ -0,0 +1,14 @@
+{
+    "version": "3.0.8",
+    "description": "Complete set of commands to define, develop, test serverless applications, and deploy them to the Alibaba Cloud.",
+    "homepage": "https://github.com/alibaba/funcraft",
+    "license": "MIT",
+    "url": "https://gosspublic.alicdn.com/fun/fun-v3.0.8-win.exe.zip",
+    "hash": "72d148c000efb16d8ec8a2355e157b4904e58da1d50dd1ee6187cb70ef516e21",
+    "pre_install": "Rename-Item \"$dir\\fun*win.exe\" 'fun.exe'",
```

Safer for wider users, as I do not know in which environments it is working.

```suggestion
    "pre_install": "Move-Item \"$dir\\fun*win.exe\" \"$dir\\fun.exe\"",
```